subworkflows/local/utils_nfcore_ascc_pipeline/main.nf

```diff
@@ -77,7 +78,15 @@ workflow PIPELINE_INITIALISATION {
         .fromList(samplesheetToList(params.input, "${projectDir}/assets/schema_input.json"))
         .map {
             sample, type_of_assembly, assembly_file ->
-                return [[ id: sample.id + '_' + type_of_assembly, assembly_type: type_of_assembly, single_end:true ], assembly_file ]
+                return [
+                    [
+                        id: sample.id + '_' + type_of_assembly,
+                        assembly_type: type_of_assembly,
+                        filter_cutoff: params.filter_cutoff,
```

Comment:
Is there a reason to put this in `meta` rather than just referencing the param in `modules.conf`?